src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -101,6 +105,8 @@ public class CommandTestUtil {
     public static final String INVALID_START_TIME_DESC = " " + PREFIX_START_TIME + "11:30"; // is after end time
     public static final String INVALID_END_TIME_DESC = " " + PREFIX_END_TIME + "11:00"; // is before start time
     public static final String INVALID_APPOINTMENT_TYPE_DESC = " " + PREFIX_TAG + "  "; // only white spaces
+    public static final String INVALID_APPOINTMENT_NOTE_DESC = " " + PREFIX_NOTE + "@@"; // non-alphanumeric
```

Comment:
In my previous PR, I removed validity checks for appointment notes - any string is acceptable as of now!